.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!---
+BEFORE YOU SUBMIT please search open/closed issues since someone might have asked the same thing before!
+-->
+
+### Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
```

Comment:
I like the idea, but would prefer to keep the template itself minimal. People usually have that tendency to either skip or not fill in the template. The `current` and `expected` behaviour are formulated in a way that should encourage people to use it. 

Time will show 💃